### ISSUE TITLE
ShulkerBoxTooltip

### ISSUE THREAD

**Issue by Madis0** [Fabulously-Optimized/fabulously-optimized#2]
**Mod name**
ShulkerBoxTooltip

**Curseforge link**
https://www.curseforge.com/minecraft/mc-mods/shulkerboxtooltip

**Modrinth link**
N/A

**Other link**
https://github.com/MisterPeModder/ShulkerBoxTooltip

**What it does**
Adds a visual tooltip for shulker boxes that show their contents.

**Why should it be in the modpack**
1.18 bundle parity, [unless it gets implemented in vanilla](https://feedback.minecraft.net/hc/en-us/community/posts/360074507051-shulker-boxes-should-have-the-new-bundle-interface) (please vote)

**Why shouldn't it be in the modpack**
Slightly less vanilla behaviour, https://github.com/MisterPeModder/ShulkerBoxTooltip/issues/43

**Categories**
Select any that match:
- [ ] Performance optimization
- [x] Graphics optimization
- [ ] New feature
- [ ] Optifine parity
- [x] Fixes a bug/dependency

**Additional details**
Edit: [bundles have been postponed](https://www.minecraft.net/en-us/article/minecraft-snapshot-21w19a), so I'll postpone this as well
Edit 2: bundles postponed again to 1.19
Edit 3: postponed again because [they didn't figure out how to hover with touchscreens](https://youtu.be/ixSlQXDOvkM?t=174)

**Comment by wafflecoffee** [comment]
Would be obsoleted by #66. FWIW, Inspecio is most likely going to have better general compatibility than SBT. On one of my own mods, it contains a custom screen handler to add some extra features to an existing vanilla screen. SBT was unable to detect shulkers within this container, but Inspecio was able to do it flawlessly. I don't know how many mods add a custom screen handler, especially not any within this modpack, but it's still worth considering.

**Comment by nsde** [comment]
Any updates on whether this mod should be added or Inspecio or neither?

**Comment by Madis0** [comment]
This basically waits on vanilla to see when and how bundles will be added to survival and in that case, will they change the shulker box tooltips as well. 

Ideally the mod would only be added to 1.18 as a parity fallback and 1.19 will have it as part of vanilla. But for that to happen, please vote on the idea linked in the OP 😉

**Comment by Madis0** [comment]
Even if bundles were implemented with the tooltips and shulker boxes didn't get them, this could be seen as cheaty and it defies vanilla expectations, possibly also someone's workflow who were used to reading text in tooltips. Therefore - rejected.
